index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```

Comment:
This is the best comment
```suggestion
</table> must be deleted
```